web/components/OnboardingOptions.tsx

```diff
@@ -0,0 +1,107 @@
+import { Flex, useBreakpointValue } from "@chakra-ui/react";
+import Select from "react-select";
+import OnboardingOptionColumn from "./OnboardingOptionColumn";
+
+function OnboardingOptions(props: {
+  options: string[];
+  singleAnswer: boolean;
+  dropdown: boolean;
+  answers: boolean[][];
+  setAnswers: (arg: boolean[][]) => void;
```

Comment:
setState, if using React's useState hook, has type `Dispatch<SetStateAction>` which can be generally preferred